app/components/TrackCard/index.js

```diff
@@ -0,0 +1,87 @@
+/**
+ *
+ * TrackCard
+ *
+ */
+
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { Card } from '@mui/material';
+import T from '@components/T';
+import If from '@components/If';
+import isEmpty from 'lodash/isEmpty';
+import Box from '@mui/material/Box';
+import CardContent from '@mui/material/CardContent';
+import CardMedia from '@mui/material/CardMedia';
+import IconButton from '@mui/material/IconButton';
+import Typography from '@mui/material/Typography';
+import PlayArrowIcon from '@mui/icons-material/PlayArrow';
+import PauseIcon from '@mui/icons-material/Pause';
+
+export function TrackCard({ collectionName, artistName, shortDescription, artworkUrl100, previewUrl }) {
+  const [playTrack, setPlayTrack] = useState(false);
+  const shortDesc =
+    shortDescription && shortDescription.length > 38 ? `${shortDescription.substring(0, 38)}...` : shortDescription;
+
+  return (
+    <Card
+      sx={{
+        display: 'flex',
```

Comment:
Instead of writing css inline , can we instead use with a styled component. This will make it easy to view and debug css later as compared to parsing through inline css